llvmlite/binding/newpassmangers.py

```diff
@@ -0,0 +1,305 @@
+from ctypes import (c_bool, c_int)
```

Comment:
No need for the brackets:

```suggestion
from ctypes import c_bool, c_int
```